bom-internal/pom.xml

```diff
@@ -61,7 +61,7 @@ POM as their parent.
         <cwlavro.version>2.0.4.7</cwlavro.version>
         <okhttp.version>4.7.0</okhttp.version>
         <slf4j.version>1.7.32</slf4j.version>
-        <swagger-ui.version>3.25.0</swagger-ui.version>
+        <swagger-ui.version>4.12.0</swagger-ui.version>
```

Comment:
This one doesn't look like it's used. I think you can optionally just remove it. But if you want to limit the scope of the change for the hotfix PR, that's OK

Reply:
Removed it